applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/server_store.h

```diff
@@ -0,0 +1,413 @@
+				    uint32_t *valid_codec_caps,
+				    struct client_supp_configs const **const client_supp_cfgs,
+				    uint8_t num_client_supp_cfgs);
+
+/**
+ * @brief	Get a server based on stream pointer.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @param[in] stream  Pointer to the stream.
+ * @param[out] server  Pointer to the server structure to fill. NULL if not found.
+ *
+ * @retval	0 Success.
+ * @retval	-ENOENT Server not found.
+ */
+int srv_store_from_stream_get(struct bt_bap_stream const *const stream,
+			      struct server_store **server);
+
+/**
+ * @brief	Get the number of endpoints in a given state across all stored servers.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @param[in] state  State to search for.
+ * @param[in] dir   Direction to filter on.
+ *
+ * @retval	Number of endpoints in the given state and direction.
+ * @retval	-ENOTCONN Server conn is NULL.
+ */
+int srv_store_all_ep_state_count(enum bt_bap_ep_state state, enum bt_audio_dir dir);
+
+/**
+ * @brief	Store the available audio context for a server based on conn dst address.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @param[in] conn  Pointer to the connection.
+ * @param[in] snk_ctx   Sink context.
+ * @param[out] src_ctx  Source context.
+ *
+ * @retval	0 Success.
+ * @retval	-EINVAL Illegal argument(s)
+ * @retval	-ENOENT Server not found.
+ */
+int srv_store_avail_context_set(struct bt_conn *conn, enum bt_audio_context snk_ctx,
+				enum bt_audio_context src_ctx);
+
+/**
+ * @brief	Store the codec capabilities for a given server based on conn dst address.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @param[in] conn  Pointer to the connection.
+ * @param[in] dir   Direction to store.
+ * @param[out] codec  Codec capabilities to store.
+ *
+ * @retval	0 Success.
+ * @retval	-EINVAL Illegal argument(s)
+ * @retval	-ENOMEM Out of memory.
+ * @retval	-ENOTCONN Server conn is NULL.
+ */
+int srv_store_codec_cap_set(struct bt_conn const *const conn, enum bt_audio_dir dir,
+			    struct bt_audio_codec_cap const *const codec);
+
+/**
+ * @brief	Get a server from the address.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @param[in] addr  Pointer to the address to find the server for.
+ * @param[out] server  Pointer to the server structure to fill. NULL if not found.
+ *
+ * @retval	0 Success.
+ * @retval	-ENOENT Server not found in storage.
+ */
+int srv_store_from_addr_get(bt_addr_le_t const *const addr, struct server_store **server);
+
+/**
+ * @brief	Find if a stored server exists based on address.
+ *
+ * @param[in] addr  Pointer to the address to find the server for.
+ *
+ * @note	srv_store_lock() must be called before accessing this function.
+ *
+ * @retval	true if the server exists, false otherwise.
```

Comment:
brief, note, param, retval -- that's the order
```suggestion
 * @note	srv_store_lock() must be called before accessing this function.
 *
 * @param[in] addr  Pointer to the address to find the server for.
 *
 * @retval	true if the server exists, false otherwise.
```